read /usr/share/mutt/mailcap as fallback by default. This allows to
set some useful defaults specifically for mutt. For example
text/html
Index: mutt-1.5.21/init.c
===================================================================
---
 init.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- init.c
+++ init.c	2019-01-02 13:25:20.795634552 +0000
@@ -3620,7 +3620,7 @@ void mutt_init (int skip_sys_rc, LIST *c
   else
   {
     /* Default search path from RFC1524 */
-    MailcapPath = safe_strdup ("~/.mailcap:" PKGDATADIR "/mailcap:" SYSCONFDIR "/mailcap:/etc/mailcap:/usr/etc/mailcap:/usr/local/etc/mailcap");
+    MailcapPath = safe_strdup ("~/.mailcap:" PKGDATADIR "/mailcap:" SYSCONFDIR "/mailcap:/usr/etc/mailcap:/usr/local/etc/mailcap:" SYSCONFDIR "/mutt/mailcap");
   }
 
   Tempdir = safe_strdup ((p = getenv ("TMPDIR")) ? p : "/tmp");
